bucket/sioyek.json

```
@@ -0,0 +1,53 @@
+{
+    "version": "1.1.0",
+    "description": "A PDF viewer designed for reading research papers and technical books",
+    "homepage": "https://sioyek.info/",
+    "license": "GPL-3.0-only",
+    "suggest": {
+        "Microsoft Visual C++ Redistributables 2019": "extras/vcredist2019"
```

```suggestion
        "Microsoft Visual C++ Redistributables": "extras/vcredist2022"
```
VCRedist 2022 now [supports all functionalities of VCRedist2015-2022](https://docs.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170#visual-studio-2015-2017-2019-and-2022)

---

Neat! Thanks